### PR TITLE
Add pundit

### PR DESCRIPTION
**Pull request by lucasenriquez**
## Summary

- [x] I have added rent policy. 
- [x] When a user tries to open another user's rents, pundit denies his access.
- [x] I have reviewed the files changed before releasing the pull request
- [x] BEFORE MERGE: I have updated the TRELLO card column before merging the pull request

## Screenshots

image

## TRELLO Card

https://trello.com/c/BOF7csS7/12-instalar-y-configurar-pundit

### CODE REVIEW

File: app/controllers/api_controller.rb [hunk 1]
```diff
@@ -19,4 +20,12 @@ def validation_error(resource)
       ]
     }, status: :bad_request
   end
+
+  rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
+
+  private
+
+  def user_not_authorized
+    render json: { error: 'You are not authorized to perform this action.' }
```
Comment (dante-wd) [inline app/controllers/api_controller.rb:29]: How about adding a test to your api_controller_spec.rb that checks for this?

File: app/controllers/rents_controller.rb [hunk 2]
```diff
@@ -25,6 +24,8 @@ def create
   def send_mail
     user = current_user
     title = @book[:title]
+    @from = @rent[:from]
```
Comment (melissaregnier) [inline app/controllers/rents_controller.rb]: you are never using this @from

File: app/controllers/rents_controller.rb [hunk 3]
```diff
@@ -25,6 +24,8 @@ def create
...
+    @to = @rent[:to]
     RentEmailWorker.perform_async(user[:email], user[:fist_name], title, @to, user[:locale])
```
Comment (melissaregnier) [inline app/controllers/rents_controller.rb]: you could simply use @rent[:to] here and avoid the variable altogether

**Review by melissaregnier — CHANGES_REQUESTED** [review]

File: app/controllers/rents_controller.rb (hunk 2 above)
Comment (lucasenriquez) [reply app/controllers/rents_controller.rb]: done

File: app/controllers/rents_controller.rb (hunk 3 above)
Comment (lucasenriquez) [reply app/controllers/rents_controller.rb]: Done

File: app/controllers/api_controller.rb (hunk 1 above)
Comment (lucasenriquez) [reply app/controllers/api_controller.rb:29]: Done

**Review by dante-wd — APPROVED** [review]

**Review by NicoFunes — APPROVED** [review]

**Pull request merged**